provision/ubuntu/kernel-next.sh

```diff
@@ -29,7 +29,7 @@ export LD_LIBRARY_PATH=$LD_LIBRARY_PATH:/usr/local/lib/
 
 # Build kernel
 cp /boot/config-`uname -r` .config
-make oldconfig && make prepare
```

Comment:
Nit: I think there's a typo in this commit description: 
> use `make olddefconfig` instead of `make olddefconfig`

Reply:
Oops, the dangers of autocompletion 😄 will push a corrected commit message.